### PR TITLE
Update tradfri.markdown to clarify install pre-requisites

### PR DESCRIPTION
**Pull request by ggravlingen**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

  - [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/tradfri.markdown [hunk 1]
```diff
@@ -38,3 +38,7 @@ Configuration variables:
 <p class='note'>
 Do not use the `api_key` variable. The key is only needed once at initial setup.
 </p>
+
+<p class='note'>
+Please make sure you have autoconf installed (```sudo apt-get install autoconf```) and setuptools (```sudo pip3 install setuptools```) if you want to use this component. Also, installing some dependencies might take considerable time (>1h) on slow devices.
```
Comment (MartinHjelmare) [inline source/_components/tradfri.markdown]: Python packages should be installed in the environment running home assistant, so use of `sudo` is environment specific and not generally recommended.

**Comment by MartinHjelmare** [comment]
If tradfri requires setuptools why shouldn't we add that as a requirement in the component? Then we wouldn't have to document it here.

**Comment by MartinHjelmare** [comment]
Wait, home assistant also requires setuptools, so how could you even start home assistant without that?

**Comment by ggravlingen** [comment]
@MartinHjelmare sorry, you are right about setuptools.

**Comment by ggravlingen** [comment]
Do I need to do anything for this to be merged? I'm uncertain what the label implies.

File: source/_components/tradfri.markdown [hunk 2]
```diff
@@ -38,3 +38,7 @@ Configuration variables:
 <p class='note'>
 Do not use the `api_key` variable. The key is only needed once at initial setup.
 </p>
+
+<p class='note'>
+Please make sure you have autoconf installed (```apt-get install autoconf```) if you want to use this component. Also, installing some dependencies might take considerable time (>1h) on slow devices. You might have to use ```sudo``` when installing autoconf.
```
Comment (MartinHjelmare) [inline source/_components/tradfri.markdown]: Shouldn't inline code be formatted using single backtics?

Comment (ggravlingen) [reply source/_components/tradfri.markdown]: Missed that part of the standards. Have changed now.

**Comment by frenck** [comment]
@ggravlingen Looks good, nice addition.
The labels do not say anything. It is just a method for us as the maintainers (and visitors as well) to see what type of PR this is.

**Pull request merged**